inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -230,4 +232,21 @@ public function maybe_trigger_recreate_table() {
 
 		delete_option( $this->db_version_key );
 	}
+
+	/**
+	 * Add error_message column and index
+	 *
+	 * @return bool
+	 */
+	protected function add_error_message_column() {
+		$error_message_column_exists = $this->column_exists( 'error_message' );
+
+		$created = true;
+
+		if ( ! $error_message_column_exists ) {
+			$created &= $this->get_db()->query( "ALTER TABLE {$this->table_name} ADD COLUMN error_message longtext NULL default NULL AFTER hash" );
```

Comment:
Enclose the table name between `` like.

```suggestion
			$created &= $this->get_db()->query( "ALTER TABLE `{$this->table_name}` ADD COLUMN error_message longtext NULL default NULL AFTER hash" );
```